packages/react-native-ui/src/hooks/useStoreState.ts

```diff
@@ -0,0 +1,12 @@
+import { useEffect, useState } from 'react';
+import store from 'store2';
```

Comment:
I don't think this store actually works, I'm realising now that its in the lastViewed hook but I think its just an artifact from me copy/pasting a bit too freely from the web code 😅

We have the storage from the getStorybookUI thats required maybe we can pass that down to the ui

Reply:
yeah the thing is that we also want tablets and other wide screen devices to work so that would be ideal